### DISCUSSION TITLE
First model performance optimization technique to test

### DISCUSSION THREAD

**Discussion by bhupali08** [bhupali08/Image-Classification-using-Deep-Learning-for-a-city-dog-show#12] [Q&A] [answered]
Hello, 

could you suggest what should be some of the first techniques that we can test to improve the model performance?

**Comment by SohamBera16** [comment, marked as answer]
I think trying to explore how to make the dataset better should be the ideal first step. let's take the best-performing current model (AlexNet) as the baseline, make an error analysis on the misclassified samples, and tinker with the data (augmentations, increasing data, reducing noise by image processing, class imbalances, etc.) to make it better so that those misclassification issues can be resolved.
